### PR TITLE
Fix typos in CREATE-NEW-SCALER.md

### PR DESCRIPTION
**Pull request by inuyasha82**
As mentioned in issue #531 there are few typos in the CREATE-NEW-SCALER.md file, this PR is just trying to fix them.

### CODE REVIEW

File: CREATE-NEW-SCALER.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 In order to developer a scaler, a developer should do the following:
 1. Download KEDA's code
-2. Define the main pieces of data that you expect the user to supply so the scaler runs properly. For example, if your scaler needs to connect to an external source based on a connection string, you expect the user to supply this connection string in the conifugration within the ScaledObject under `trigger`. This data will be passed to your constructing function as map[string]string.
+2. Define the main pieces of data that you expect the user to supply so the scaler runs properly. For example, if your scaler needs to connect to an external source based on a connection string, you expect the user to supply this connection string in the confguration within the ScaledObject under `trigger`. This data will be passed to your constructing function as map[string]string.
```
Comment (tomkerkhove) [inline CREATE-NEW-SCALER.md]:
```suggestion
2. Define the main pieces of data that you expect the user to supply so the scaler runs properly. For example, if your scaler needs to connect to an external source based on a connection string, you expect the user to supply this connection string in the configuration within the ScaledObject under `trigger`. This data will be passed to your constructing function as map[string]string.
```

Comment (inuyasha82) [reply CREATE-NEW-SCALER.md]: oh lol i fixed a typo with another typo! 🤦‍♂

Comment (tomkerkhove) [reply CREATE-NEW-SCALER.md]: No worries, that's what reviews are for!

**Comment by ahmelsayed** [comment]
Thank you @inuyasha82!

**Pull request merged**